packages/@stylexjs/babel-plugin/src/shared/preprocess-rules/basic-validation.js

```diff
@@ -58,7 +58,14 @@ function validateConditionalStyles(
 ): void {
   for (const key in val) {
     const v = val[key];
-    if (!(key.startsWith('@') || key.startsWith(':') || key === 'default')) {
+    if (
+      !(
+        key.startsWith('@') ||
+        key.startsWith(':') ||
+        key.startsWith('var(--') ||
+        key === 'default'
+      )
+    ) {
       throw new Error(messages.INVALID_PSEUDO_OR_AT_RULE);
```

Comment:
Here too - if I look at the condition, it's unclear why starting not with `var(--` should result in `INVALID_PSEUDO_OR_AT_RULE` being thrown